data_generators/job_description_scraper.py

```diff
@@ -564,6 +564,89 @@ def combine_individual_datasets(self, folder_date):
 
         print('Successfully saved combined dataset!')
 
+        return
+
+    def add_job_titles(self, job_list):
+        """
+        Add more job titles to search through.
+
+        Args:
+            job_list (list): A list of job titles in HTML
+            format for the scraper to search through.
+            Remember that empty spaces in words need to
+            be replaced with "%20" to stay HTML compliant.
+                ex: ["web%20developer","backend%20engineer"]
```

Comment:
Can the space conversion to `%20` be automated so our users don't need to remember this? Currently it breaks encapsulation in the sense that our http requirements are leaking. Look for something called url encoding, I'm pretty sure Python has a prebuilt solution for this in the standard library.

Reply:
Done!